coriolisclient/v1/endpoints.py

```diff
@@ -67,3 +68,20 @@ def validate_connection(self, endpoint):
             json={'validate-connection': None}).json()
         validate_data = data["validate-connection"]
         return validate_data.get("valid"), validate_data.get("message")
+
+    def get_endpoint_id_for_name(self, endpoint):
+        if validate_uuid(endpoint):
+            return endpoint
+        else:
+            return self._get_endpoint_id_for_name(endpoint)
+
+    def _get_endpoint_id_for_name(self, endpoint_name):
+        """ Gets the UUID of the endpoint from the parsed name """
+        obj_list = self.list()
+        id_matches = [n.id for n in obj_list if n.name == endpoint_name]
+        if len(id_matches) > 1:
```

Comment:
NIT: Please declare a separate var for `len(id_matches)` considering you're using it in two places.
This'll save some electron charge which the universe would never be getting back otherwise.